src/dackar/utils/mbse/customMBSEparser.py

```diff
@@ -0,0 +1,177 @@
+# Copyright 2024, Battelle Energy Alliance, LLC  ALL RIGHTS RESERVED
+
+"""
+Created on February, 2024
+
+@author: mandd
+"""
+
+# External Imports
+import pandas as pd
+
+class customMBSEobject(object):
+    """
+        Class designed to process the a custom MBSE model from file.
+    """
+    def __init__(self, nodes_filename, edges_filename):
+        """
+        Initialization method for the custom MBSE model class
+
+        Args:
+
+            nodes_filename: file, file in .csv format containing all nodes 
+            edges_filename: file, file in .csv format containing all edges 
+
+        Returns:
+
+            None
+        """
+        self.nodes_filename = nodes_filename
+        self.edges_filename = edges_filename
+        self.entities = []
+        self.listIDs = []
```

Comment:
I would recommend we do not mix the naming rules, either use cameBack or snake case.

Reply:
fixed